DfE.FindInformationAcademiesTrusts/Pages/Trusts/Overview.cshtml

```diff
@@ -0,0 +1,97 @@
+﻿@page
+@model OverviewModel
+
+@{
+  Layout = "_TrustLayout";
+}
+
+<section>
+  <div class="govuk-!-margin-top-9 govuk-!-margin-bottom-1">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+        <section class="govuk-summary-card" aria-labelledby="trust-information">
```

Comment:
As mentioned, could we remove the aria-labelled by on the sections, in line with the example for the [GOV.UK summary card](https://design-system.service.gov.uk/components/summary-list/card-with-actions/index.html) and our [trust details page](https://github.com/DFE-Digital/find-information-about-academies-and-trusts/blob/main/DfE.FindInformationAcademiesTrusts/Pages/Trusts/Details.cshtml)